docs/doc-generator/www-creator.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+HOME=`pwd`
```

Comment:
This file should be executed through maven. Maybe we can add a maven build profile section and within that a maven exec plugin to run this script. 

```xml
    <profiles>
        <profile>
            <id>doc-site-serve</id>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.codehaus.mojo</groupId>
                        <artifactId>exec-maven-plugin</artifactId>
                        <executions>
                            <execution>
                                <id>site</id>
                                <goals>
                                    <goal>exec</goal>
                                </goals>
                            </execution>
                        </executions>
                        <configuration>
                            <executable>${project-home}/docs/doc-generator/www-creator.${script.extension}</executable>
                        </configuration>
                    </plugin>

                </plugins>
            </build>
        </profile>
    </profiles>

```

```bash
mvn exec:exec -P doc-site-serve
```

Reply:
As offline discussion removed the `www-creator.sh` and added steps to build the site in `README.md`.